.env.development

```diff
@@ -1,4 +1,4 @@
-REACT_APP_DAS_HOST=https://develop2.pamdas.org
+REACT_APP_DAS_HOST=https://develop.pamdas.org
```

Comment:
I guess it should be `root.develop` by default now?

Reply:
fixed